README.md

```diff
@@ -602,6 +602,37 @@ The response body includes:
 }
 ```
 
+## Public Metrics API
+
+Public metrics are high-level aggregations of study data, suitable to expose
+in public dashboards. Public metrics are grouped into metric sets, which share a
+common schema. One metric set will correspond to a live view of the RDR data,
+while others will be fixed snapshots taken during curated data releases.
+
+#### `GET /PublicMetricSets`
+
+List all available metric sets and their IDs.
+
+The response body includes:
+
+* `metricSets`: an array of metric sets
+
+```
+{
+  "metricSets": [
+    {"id": "live"},
```

Comment:
I'm wondering if for the sake of extensibility, we should have two parts to the ID: a type and a version, effectively.

So you might have { "type": "participants", "version": "live" } (or maybe "id": "live"). 

This would leave room for other types of metrics in future.

Reply:
I'd prefer just an ID to a composite, but I added a `type` field. In practice, we'll be defining this values in code so we can choose whatever format we prefer for these IDs (including a type probably makes sense). I just find composite IDs annoying to pass around as it's almost an implementation detail of the resource ID that clients shouldn't necessarily need to parse or reconstruct. WDYT?

PS: my true preference on API IDs (per the Cloud style guide) is using a singular resource `name` to refer to things, e.g. `foos/<food_id>/bars/<bar_id>`; APIs would accept `bar_name: "foos/<food_id>/bars/<bar_id>"` to refer to this. But I won't push for this since it doesn't work very well with Swagger unfortunately.